packages/xo-web/src/common/intl/messages.js

```diff
@@ -803,6 +803,13 @@ const messages = {
   srShared: 'Shared',
   srNotShared: 'Not shared',
   pbdNoSr: 'No storage detected',
+  deletePbdModalTitle: 'Delete PBD',
+  deletePbdModalMessage:
+    'Are you sure you want to forget this PBD? This will disconnect the SR from all their hosts by removing their links to them (PBDs)',
+  deletePbdsModalTitle: 'Delete PBD{nPbds, plural, one {} other {s}}',
+  deletePbdsModalMessage:
+    'Are you sure you want to forget {nPbds, number} PBD{nPbds, plural, one {} other {s}}? This will disconnect the SR from all their hosts by removing their links to them (PBDs)',
```

Comment:
Are you sure you want to forget N hosts?
This will disconnect the SR from these hosts by removing the links between the SR and the hosts (PBDs).